superset/jinja_context.py

```diff
@@ -40,6 +41,16 @@ def url_param(param, default=None):
     return request.args.get(param, default)
 
 
+def current_user_id():
+    """The id of the user who is currently logged in"""
+    return g.user.id
```

Comment:
is it possible that `g.user` won't be defined at some point? should we return null in this case and catch the exception?